docs/API.md

```diff
@@ -492,6 +492,19 @@ The board list endpoint supports setting an `If-Modified-Since` header to limit
 
 ##### 200 Success
 
+### POST /boards/{boardId}/stacks/{stackId}/clone - Clone a stack
+
+#### Request parameters
+
+| Parameter | Type    | Description                              |
+| --------- | ------- | ---------------------------------------- |
+| boardId   | Integer | The id of the board the stack belongs to |
+| stackId   | Integer | The id of the stack                      |
+
+#### Response
+
+##### 200 Success
```

Comment:
Let's also describe the response here a bit that it will return an object of the new stack containing the new cards as well.

Ideally please add some example JSON of the response.